eth/downloader/skeleton.go

```diff
@@ -221,22 +221,26 @@ type skeleton struct {
 	terminate  chan chan error  // Termination channel to abort sync
 	terminated chan struct{}    // Channel to signal that the syncer is dead
 
+	// For XLayer, get legacyXLayerBlock from chainConfig
+	legacyXLayerBlock uint64
```

Comment:
Maybe we can use more generic name

```suggestion
	cutOffBlock uint64
```